utils/email.py

```diff
@@ -0,0 +1,38 @@
+
+
+# import necessary packages
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from email.mime.image import MIMEImage
+
+import smtplib
+from string import Template
+import os
+
+
+def send_mail(s: smtplib.SMTP, message_template: Template, name: str,
+              dest_email: str, src_email: str, mail_subject:str, image_file: str):
+    msg = MIMEMultipart()       # create a message
+
+    # add in the actual person name to the message template
+    message = message_template.substitute(PERSON_NAME=name.title())
+
+    # setup the parameters of the message
+    msg['From'] = src_email
+    msg['To'] = dest_email
+    msg['Subject'] = mail_subject
+
```

Comment:
Add CC and BCC, and set default parameter values as `None.`

Reply:
added `cc` and `bcc` parameters, these parameters are directly added to plaintext version of smtp mail, so there are no hidden default parameters to set `none`